app/services/rapid_ready_for_decision/views/hypertension_pdf.erb

```diff
@@ -0,0 +1,139 @@
+Hypertension Rapid Ready for Decision | Claim for Increase <br />
+<h1>
+  VHA Hypertension Data Summary for <br/>
+    <%= patient_name %>
+</h1>
+<i>Generated automatically on <%= generated_at %></i>
+
+<p>
+  <% if blood_pressure_data? %>
+    <h4>One Year of Blood Pressure History</h4>
+  <% else %>
+    <h4>No blood pressure records found</h4>
+  <% end %>
+</p>
+
+<i>VHA records searched from <%= start_date %> to <%= end_date %></i> <br/>
+<i>All VAMC locations using VistA/CAPRI were checked</i>
+
+<p> <br /> </p>
+
+<% if blood_pressure_data? %>
+  <p>Blood pressure is shown as systolic/diastolic.</p>
+<% end %>
+
+<p> <br /> </p>
+
+<% @blood_pressure_data.each do |bp| %>
+  <% systolic = bp[:systolic]['value'].round %>
+  <% diastolic = bp[:diastolic]['value'].round %>
+
+  <p>
+    <b>Blood pressure: <%= systolic %>/<%= diastolic %></b> <br />
+    Taken on: <%= bp[:effectiveDateTime].to_date.strftime('%m/%d/%Y') %>
+      at <%= Time.iso8601(bp[:effectiveDateTime]).strftime('%H:%M %Z') %> <br />
+    Location: <%= bp[:organization] || 'Unknown' %>
+  </p>
+<% end %>
+
+<h4>Hypertension Rating Schedule</h4>
```

Comment:
Remember to wrap this in an `if` block if you decide to generalize this template.